charts/tekton-pipelines/values.yaml

```diff
@@ -90,6 +92,11 @@ buildpacks:
               echo "Preparing for buildpack build"
               cp -rf {project.toml,nginx.conf,buildpack.yml,nginx.d,.nginx.d,nginx.*,httpd.conf,.http.d,httpd.d,Procfile} $(params.source_subpath)  2>/dev/null
               chown -R $(params.user_id):$(params.group_id) $(params.source_subpath)
+
+              # copy built dir to workspace for further usage in sentry-release task (source maps release)
+              echo "Save source maps"
```

Comment:
so how do we generate the source map? I haven't seen it?

Reply:
let's extract this into its own step where we
- generate the source map (called via npm script command)
- store it where it needs to be stored

Reply:
Source maps are generated with `npm run build:dev` command there is no other way to do this. So frontend projects define `sourcemaps: true` param in their repo, then `npm run build:dev` command builds static with source maps it and we just save it to `workspace` here to process later in `sentry-release` step

Reply:
So do we expect `npm run build:$(params.environment)` to have steps defined in package.json to build the sourcemap?

Can you show me how that command look like?

Reply:
Frontend frameworks like Angular or others should have configs with `sourceMap: true` in for example angular.json files (https://github.com/saritasa-nest/allstar-elevator-frontend/blob/develop/angular.json#L65), that's how frontend tells to build static with special `*.js.map` files (they are needed for developers debug purpose and Sentry to map minified js codes to source codes).

So basically source maps are created in `npm run build:dev` command (if they have a `sourceMap: true` flag in corresponding js codes) and here we just save them in Tekton `workspace` for later usage in `sentry-release` task.